content/guides/embed/ui-elements/theming-styling.md

```diff
@@ -0,0 +1,122 @@
+---
+rank: 11
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+  - embed/ui-elements/explorer
+  - embed/ui-elements/uploader
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+---
+<!--alex ignore -->
+# Theming & styling for Box UI Elements
+
+With theming & styling for Box UI Elements you can customize the look of the
+embedded Box component to your enterprise's requirements.
+<!--alex ignore -->
+The tool provides a real-time live preview for an easy adjustment of the theme.
```

Comment:
We don't have a tool for that, and there is not real-time preview. This is a future plan. For now you have to update the code directly.